samples/sampler/stories/default/About.js

```diff
@@ -95,7 +95,7 @@ export const ATourOfSampler = (args) => (
 			</b>{' '}
 			for a demonstration
 		</HintDialog>
-		<HintDialog style={{bottom: riSafe(edgeDotKeepout), left: 114, top: 582}} length={348}>
+		<HintDialog style={{bottom: riSafe(edgeDotKeepout), left: 114}} length={156}>
```

Comment:
How does come the 156 value? 
Why only bottom wrapped by riSafe? Why left is not wrapped?

Reply:
156 is a heuristic value that makes hint dialogs do not overlap each other.
I refered prior difference of the hint dialog length between line 88 and line 98.
[bottom] should be wapped by riSafe because the size of hint dialog changes when you expand the side bar left to right.
But [left] position should not because the position of [Actions] and [Controls] buttons maintains their size.